roles/edpm_ovn/tasks/configure.yml

```diff
@@ -89,5 +89,23 @@
         setype: "container_file_t"
         mode: "0644"
       with_items:
-      - {"src": "rootwrap.conf.j2", "dest": "rootwrap.conf"}
-      - {"src": "neutron-ovn-metadata-agent.conf.j2", "dest": "neutron-ovn-metadata-agent.conf"}
+      - {"src": "rootwrap.conf.j2", "dest": "01-rootwrap.conf"}
```

Comment:
for rootwrap would also need to adjust edpm_ovn_metadata_agent_metadata_agent_agent_root_helper to new location, basically missed in the previous commit.

Also can clean unused edpm_ovn_metadata_agent_neutron_config and it dependencies here or in seperate PR.